### PR TITLE
Add files via upload - Item for Sellers page Systems, Tools and Registration sellers guide

### PR DESCRIPTION
**Pull request by smteague**

### CODE REVIEW

**Comment by smteague** [comment]
@weiwang-gsa Hello Wei, in this patch I uploaded a pdf and then linked to in from the Resources page. When I looked at the sandbox page for this branch and opened the file from the Resources page, it actually opened in a github frame, which I don't think it's supposed to do. Maybe it's because the file itself hasn't been pushed live, but I think I might have done something wrong. Thank you for taking a look at it when you can.

File: src/content/resources/index.mdx [hunk 1]
```diff
@@ -27,6 +27,7 @@ To see a comprehensive overview of OASIS+ Apparent Contractor Awardees across th
 | [Symphony Task Order System - How to use the portal](https://industrysupport.apexlogic.com/support/solutions/articles/35000224610-how-to-use-the-oasis-symphony-portal) | Provides instructions for getting started with Symphony. |
 | [Symphony Task Order System - How to build an offer](https://industrysupport.apexlogic.com/support/solutions/articles/35000222331-how-to-build-your-offer-for-oasis-) | Provides instructions for building an offer in Symphony. |
 | [Frequently Asked Questions](https://industrysupport.apexlogic.com/support/solutions/articles/35000223291-oasis-faq) | Frequently asked questions about Symphony. |
+| [OASIS+ Systems, Tools, and Registration Sellers Guide](https://github.com/GSA/oasis-plus/blob/smteague-patch-2/public/files/OASIS%2B_Systems%2C%20Tools%2C%20and%20Registration_Sellers%20Guide_%20Sep%202024%20(508).pdf) | A guide to the various registrations and tools required to manage your contract to help your company be in compliance with the OASIS+ program requirements. |
```
Comment (weiwang-gsa) [inline src/content/resources/index.mdx:30]: @smteague when we uploaded file to public folder, it is not external link, we can't use the github branch link as the link, you should remove http part all the way util you reached the "/files/", like this
.  when you `[OASIS+ Systems, Tools, and Registration Sellers Guide](/files/OASIS%2B_Systems%2C%20Tools%2C%20and%20Registration_Sellers%20Guide_%20Sep%202024%20(508).pdf)`, there is reference in the [how to doc](https://docs.google.com/document/d/1UgbWZZrSoaZpejDhbOkbiZLYfALajJDnKIjbZDPizRM/edit), you can give a try, and see if it resolve your issue, please ping me when you need help, I can help you correct it

**Review by weiwang-gsa — CHANGES_REQUESTED** [review]
@smte

**Review by weiwang-gsa — COMMENTED** [review]
The link itself looks perfect!! Just same suggest as the other PR, for the internal file link, maybe better to put file type indication in the link text, so person with vision impairment know the link is not link to a web page. USWDS suggest sth like below:

image

**Review by weiwang-gsa — APPROVED** [review]
Looks great to me!

**Pull request merged**